components/sprayproxy/production/kustomization.yaml

```diff
@@ -2,13 +2,13 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
   - ../base
-  - https://github.com/redhat-appstudio/sprayproxy/config?ref=8a03dffd427c35646ba6aca73ee17ec68fd356c2
+  - https://github.com/redhat-appstudio/sprayproxy/config?ref=c6f8bf26007c140d51326760e16eb52ae5072e3b
   - pipelines-as-code-secret.yaml
 
 images:
   - name: ko://github.com/redhat-appstudio/sprayproxy
     newName: quay.io/redhat-appstudio/sprayproxy
-    newTag: 8a03dffd427c35646ba6aca73ee17ec68fd356c2
+    newTag: c6f8bf26007c140d51326760e16eb52ae5072e3b
```

Comment:
I see this tag at https://quay.io/repository/redhat-appstudio/sprayproxy?tab=tags